python/pacemaker/_cts/logging.py

```diff
@@ -1,23 +1,29 @@
-""" Logging classes for Pacemaker's Cluster Test Suite (CTS)
-"""
+""" Logging classes for Pacemaker's Cluster Test Suite (CTS) """
 
-__copyright__ = "Copyright 2014-2020 the Pacemaker project contributors"
-__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+__all__ = ["LogFactory"]
+__copyright__ = "Copyright 2014-2023 the Pacemaker project contributors"
+__license__ = "LGPLv2.1+"
```

Comment:
This was GPL. Also I prefer the full license line for source code files

Reply:
I thought we had previously decided that everything in the python directory should be LGPL.

Reply:
Public stuff definitely, but internal stuff doesn't matter, and we can't unilaterally relicense existing code -- we'd have to contact everyone in the git history for permission